updating/updating_a_cluster/updating-cluster-cli.adoc

```diff
@@ -57,6 +57,9 @@ include::modules/updating-sno.adoc[leveloffset=+1]
 // Updating a cluster by using the CLI
 include::modules/update-upgrading-cli.adoc[leveloffset=+1]
 
+//Introducing oc adm upgrade recommend - Tech Preview
```

Comment:
Ideally, match the comment to the actual title. Not required but helpful usually :)